src/active-response/utilities.c

```diff
@@ -0,0 +1,89 @@
+
+
+#include "utilities.h"
+#include <stddef.h>
+#include "active_responses.h"
+
+
+
+char* splitStrFromCharDelimiter(const char * output_buf, const char delimiter, char * strBefore, char *strAfter){
+    char* retVal = NULL;
+    char *pos = NULL;
+
+    if (output_buf != NULL) {
+        pos = strchr(output_buf, delimiter);
+        retVal = pos;
+        if (strBefore != NULL){
+            strncpy(strBefore, output_buf, pos - output_buf);
+        }
+        if (strAfter != NULL){
+            strncpy(strAfter, pos + 1, strlen(pos)); 
+        }
+
+    }
+    return  retVal;
+}
+
+int isEnabledFromPattern(const char * output_buf, const char * str_pattern_1, const char * str_pattern_2){
```

Comment:
Move this function to `src/active-response/active_responses.c`

Reply:
Use the function from the macOS PR: https://github.com/wazuh/wazuh/pull/14962/files#diff-8c46d6aa3fa2cb1de063c4b912f020b5f74722352fc96e486815b088f4d88474R241 that is already reviewed and approved

Reply:
You will have to delete it from the `src/active-response/firewalls/pf.c` file in the macOS PR later and use the one from `src/active-response/active_responses.c`